phone-book/src/keypad.js

```diff
@@ -0,0 +1,308 @@
+
+                ${this.callModeKeypad()}
+
+            </div>
+        `;
+    }
+
+    callTargetComponent() {
+        return `
+            <div class="row">
+                <span class="call-target">(098) 76-54-321</span>
+                <span class="phone-code">(Kuivstar)</span>
+            </div>
+        `
+    }
+
+    callAnimationComponent() {
+        return `
+            <div class="row">
+                <span id="load-point-1" class="fas fa-circle load-point"></span>
+                <span id="load-point-2" class="fas fa-circle load-point"></span>
+                <span id="load-point-3" class="fas fa-circle load-point"></span>
+                <span id="load-point-4" class="fas fa-circle load-point"></span>
+                <span id="load-point-5" class="fas fa-circle load-point"></span>
+            </div>
+        `
+    }
+
+    callModeKeypad() {
+        const callModeKeypadTemplate = document.createElement('main');
+        callModeKeypadTemplate.classList.add('container');
+        callModeKeypadTemplate.classList.add('call-mode-keypad-block');
+
+        const row = document.createElement('div');
+        row.classList.add('row');
+
+        let counter = 0;
+
+        callModeButtons.forEach(buttonInfo => {
+            if(counter === 3) {
+                callModeKeypadTemplate.innerHTML += row.outerHTML;
+                row.innerHTML = '';
+            }
+
+            const callButton = `
+                <div class="wraper-for-call-mode-btn">
+                    <button class="call-mode-btn"><span class="fas ${buttonInfo.className}"></span></button>
+                    <span class="call-btn-description">${buttonInfo.description}</span>
+                </div>
+            `;
+
+            row.innerHTML += callButton;
+            counter++;
+        });
+
+        callModeKeypadTemplate.innerHTML += row.outerHTML;
+
+        const toCallOffButton = `
+            <div class="row">
+                <div class="wraper-for-call-mode-btn">
+                    <button id="call-off-btn" class="call-mode-btn call-off"><span class="fas fa-phone"></span></button>
+                </div>
+            </div>
+        `;
+
+        callModeKeypadTemplate.innerHTML += toCallOffButton;
+
+        return callModeKeypadTemplate.outerHTML;
+    }
+
+    render() {
+        document.body.innerHTML += this.callModeTemplate;
+    }
+
+    remove() {
+        const currentPage = document.getElementById('call-mode');
+        document.body.removeChild(currentPage);
+    }
+}    
+
+
+const keypadModeButtons = [
+    {
+        first: '1',
+        second: '2',
+        third: '3'
+    },
+    {
+        first: '4',
+        second: '5',
+        third: '6'
+    },
+    {
+        first: '7',
+        second: '8',
+        third: '9'
+    },
+    {
+        first: '*',
+        second: '0',
+        third: '#'
+    }
+];
+
+class KeypadMode{
+    constructor() {
+        this.keypadModeTemplate = `
```

Comment:
I believe we don't need keypadModeTemplate at all, please move such code to render method